torch/csrc/jit/tensorexpr/ir_simplifier.cpp

```diff
@@ -1871,6 +1871,61 @@ Stmt* TermExpander::mutate(const Free* v) {
   return new Free(buffer_var_new);
 }
 
+Stmt* TermExpander::fuseSyncThreads(Block* block) {
```

Comment:
Is `fuseSyncThreads` called anywhere?

Reply:
Yes its called in the visitor for Block immediately below.